src/scripts/components/VideoForm.js

```diff
@@ -228,16 +228,16 @@ class VideoForm extends Component<Props, Object> {
 
     // The restart button is just for convenicene, for testing
     let restartButton
-    if (process.env.NODE_ENV === 'development') {
-      restartButton = (
-        <div>
-          Use this for testing (this will not be visible in production)
-          <input type="file" onChange={this.onFileChosen} />
-        </div>
-      )
-    } else {
-      restartButton = ''
-    }
+    // if (process.env.NODE_ENV === 'development') {
```

Comment:
I commented this out so that it does not render by default. I am against having "fake" pieces of UI hard-coded in as it makes it more difficult to add/modify existing UI elements 😄. Can uncomment this if needed though